core/templates/dev/head/pages/exploration_editor/ExplorationStatesServiceSpec.ts

```diff
@@ -148,15 +148,15 @@ describe('ExplorationStatesService', function() {
       });
     });
 
-    describe('.registerOnStateAnswerGroupsSaved', function() {
+    describe('.registerOnStateInteractionSaved', function() {
       beforeEach(function() {
         spyOn(this.cls, 'editStateProperty');
       });
 
       it('callsback when answer groups of a state are saved', function() {
         var callbackSpy = jasmine.createSpy('callback');
 
-        this.ess.registerOnStateAnswerGroupsSavedCallback(callbackSpy);
+        this.ess.registerOnStateInteractionSavedCallback(callbackSpy);
```

Comment:
Can we add the test for other callbacks as well? Currently, it tests only the `saveInteractionAnswerGroups` function.

Reply:
Ah, the rest of the callbacks have already been tested later on in the file.